iina/MiniPlayerWindowController.swift

```diff
@@ -338,57 +301,31 @@ class MiniPlayerWindowController: NSWindowController, NSWindowDelegate, NSPopove
   }
 
   // MARK: - Sync UI with playback
-
-  func updatePlayButtonState(_ state: NSControl.StateValue) {
-    guard loaded else { return }
-    playButton.state = state
-  }
-
-  func updatePlayTime(withDuration: Bool, andProgressBar: Bool) {
-    guard loaded else { return }
-    guard let duration = player.info.videoDuration, let pos = player.info.videoPosition else {
-      Logger.fatal("video info not available")
-    }
-    let percentage = (pos.second / duration.second) * 100
-    leftLabel.stringValue = pos.stringRepresentation
-    rightLabel.updateText(with: duration, given: pos)
-    if andProgressBar {
-      playSlider.doubleValue = percentage
-      if #available(macOS 10.12.2, *) {
-        player.touchBarSupport.touchBarPlaySlider?.setDoubleValueSafely(percentage)
-        player.touchBarSupport.touchBarPosLabels.forEach { $0.updateText(with: duration, given: pos) }
-      }
-    }
-  }
-
   @objc
-  func updateTrack() {
-    DispatchQueue.main.async {
-      let (mediaTitle, mediaAlbum, mediaArtist) = self.player.getMusicMetadata()
-      self.titleLabel.stringValue = mediaTitle
-      self.window?.title = mediaTitle
-      // hide artist & album label when info not available
-      if mediaArtist.isEmpty && mediaAlbum.isEmpty {
-        self.titleLabelTopConstraint.constant = 6 + 10
-        self.artistAlbumLabel.stringValue = ""
+  override func updateTitle() {
+    let (mediaTitle, mediaAlbum, mediaArtist) = self.player.getMusicMetadata()
```

Comment:
I don't think we need `self` here either?

Reply:
Yeah, this was surrounded by DispatchQueue, forgot to remove those `self.`

Reply:
Since we already have a preference key with this string, I think we should leave it for now :(